sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/FcdaCandidatesHelper.java

```diff
@@ -0,0 +1,105 @@
+// SPDX-FileCopyrightText: 2022 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.util;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import org.apache.commons.lang3.StringUtils;
+import org.lfenergy.compas.sct.commons.dto.ResumedDataTemplate;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public enum FcdaCandidatesHelper implements Predicate<ResumedDataTemplate> {
```

Comment:
not sure this is an enum, I think this class is the `FcdaCandidate` that you created lower